docs/nadva/nadva-intro.mdx

```diff
@@ -8,7 +8,7 @@ sidebar_position: 1
 > 'Nadva' — isn't that 'Avdan' backward?
 
 
-*Nadva* is AvdanOS' display server. Nadva is planned to be a session [Wayland compositor](https://wayland.freedesktop.org).
+*Nadva* is AvdanOS' desktop environment which uses wayland as it's display server. Nadva is planned to be a session [Wayland compositor](https://wayland.freedesktop.org).
```

Comment:
~~desktop environment~~ 
You mean window manager?

Reply:
No what

Reply:
Nadva is the DE and the WM but it's better to say it's only the de that uses Wayland compositor or something like that

Reply:
WM != DE; Desktop environment = Default apps + Themes + Window manager (Wayland compositor) + some other stuff. WM is only a part of DE and that doesn't mean they are the same. 

Nadva is the WM or DE, I'm kinda already lost. One thing I'm sure, it's not both, not DE ***AND*** WM. It most be the DE ***OR*** only the WM. I thought it's a window manager because the PR says 

> Nadva is planned to be a session [Wayland compositor](https://wayland.freedesktop.org).

Reply:
andddd i need to find out who wrote that